src/util/PackageInfo.ts

```diff
@@ -1,13 +0,0 @@
-
-let BridgeVersion: string;
```

Comment:
This is all implemented by the upstream library now, so no need to hang onto this.